flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/filesystem/PartitionLoader.java

```diff
@@ -113,12 +113,7 @@ private void renameFiles(List<Path> srcDirs, Path destDir) throws Exception {
 					for (FileStatus srcFile : srcFiles) {
 						Path srcPath = srcFile.getPath();
 						Path destPath = new Path(destDir, srcPath.getName());
-						int count = 1;
-						while (!fs.rename(srcPath, destPath)) {
-							String name = srcPath.getName() + "_copy_" + count;
-							destPath = new Path(destDir, name);
-							count++;
-						}
+						fs.rename(srcPath, destPath);
```

Comment:
Here seems inconsistent with this method's comment: rename does not delete the dest path if it exists on HDFS. Instead, it will keep the srcPath and dest Path unchanged.

Reply:
The comment is out of date, the overwrite is occurred in `overwrite(Path)` method instead of here.
Now it is consistent with `HadoopRecoverableFsDataOutputStream.commit`.
I will update comments.